BrowserKit/Sources/MenuKit/MenuRedesign/MenuRedesignTableView.swift

```diff
@@ -159,11 +174,11 @@ final class MenuRedesignTableView: UIView,
     }
 
     func reloadTableView(with data: [MenuSection]) {
-        // We ignore first section because it is handled in MenuCollectionView
-        if let firstSection = data.first, firstSection.isTopTabsSection {
+        // We ignore first section because it is handled in MenuSquaresViewContentCell
```

Comment:
nit: I think the comment should be updated. We don't ignore the section just handled independently